### ISSUE TITLE
COCO-FUNIT:Few-Shot Unsupervised Image Translation with a Content Conditioned Style Encoder 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#48]
# TL;DR
A problem with FUNIT, a method of style transformation with Few-shot, is that some features of the content image are missing. To solve this problem, they prevented the missing features by inserting content image feature into the Style Encoder.
![image](https://user-images.githubusercontent.com/24524018/90499067-0a1fa480-e184-11ea-9f2e-61fb6ee84b0c.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.07431

# Submission Dates(yyyy/mm/dd)
2020/07/29

# Authors and institutions

Kuniaki Saito, Kate Saenko, Ming-Yu Liu
- Boston University 
- Nvidia

# Methods

# Results
![image](https://user-images.githubusercontent.com/24524018/90499082-0ee45880-e184-11ea-9dfe-335ecbce283f.png)

# Comments
